backend/core/src/shared/email/email.service.ts

```diff
@@ -60,6 +58,25 @@ export class EmailService {
     )
   }
 
+  private async loadTranslationsForUser(user: User) {
+    const language = user.language || Language.en
+    const jurisdiction = await this.jurisdictionResolverService.getJurisdiction()
+    void (await this.loadTranslations(jurisdiction, language))
+  }
+
+  public async changeEmail(user: User, appUrl: string, confirmationUrl: string, newEmail: string) {
+    await this.loadTranslationsForUser(user)
+    await this.send(
+      newEmail,
+      "Bloom email change request",
```

Comment:
Is this "Bloom email change request" the subject line? Does that not get translated?

Reply:
It does not, no. I opened another task for addressing email strings: https://github.com/CityOfDetroit/bloom/issues/724